set.go

```diff
@@ -268,6 +268,12 @@ type Set struct {
 	KeyByteOrder binaryutil.ByteOrder
 }
 
+type SetElementsInfo struct {
```

Comment:
What’s the relation between this part of the change and the rest of it? Could the set changes go into a separate commit / PR?

Reply:
Removed from the PR, I will propose this change as part of another PR. Thank You